pos_to_weight_by_product_uom/models/product_template.py

```diff
@@ -0,0 +1,15 @@
+# Copyright (C) 2017 - Today: GRAP (http://www.grap.coop)
+# @author: Quentin DUPONT <quentin.dupont@grap.coop>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import api, models
+
+
+class ProductTemplate(models.Model):
+    _inherit = 'product.template'
+
+    @api.onchange('uom_id')
+    def _onchange_uom_id(self):
```

Comment:
Why onchange instead of api.depends or related field ?

Reply:
The objective of these feature is to propose a value for 'to_weight', when user is creating a product. not to force it. Maybe some products can be with unit 'Kilo', but will be only sold by 'kilo', without the possibility to weight it.